uaclient/api/u/pro/attach/auto/full_auto_attach/v1.py

```diff
@@ -66,9 +66,29 @@ def full_auto_attach(options: FullAutoAttachOptions) -> FullAutoAttachResult:
     return _full_auto_attach(options, UAConfig(root_mode=True))
 
 
-def _full_auto_attach(options: FullAutoAttachOptions, cfg: UAConfig):
+def _full_auto_attach(
```

Comment:
Should we add an unittest where we fail to acquire the lock ?

Reply:
Good idea!